projects/RabbitMQ.Client/client/impl/RabbitMQActivitySource.cs

```diff
@@ -0,0 +1,290 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Sockets;
+using System.Reflection;
+using System.Text;
+using RabbitMQ.Client.Events;
+using RabbitMQ.Client.Impl;
+
+namespace RabbitMQ.Client
+{
+    public static class RabbitMQActivitySource
+    {
+        // These constants are defined in the OpenTelemetry specification:
```

Comment:
nit: Would you point to a specific version, as these conventions are not stable ?

Reply:
cc @stebet ?